specifyweb/frontend/js_src/lib/localization/workbench.ts

```diff
@@ -1349,6 +1349,9 @@ export const wbText = createDictionary({
     'uk-ua': 'Набір даних',
     'de-ch': 'Datensatz',
   },
+  dataSetTimestampUploaded: {
+    'en-us': 'Timestamp Uploaded',
```

Comment:
@maxpatiiuk Should this be a common string since this may be useful outside of the WorkBench? I couldn't think of another context that this could be used in yet (the batch attachment uploading tool actually shows timestamp modified, not uploaded) 

<img width="794" alt="Screenshot 2024-01-26 at 9 54 16 PM" src="https://github.com/specify/specify7/assets/37256050/6055e7b1-4ab8-404c-a507-c928bf0e551e">